client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -316,6 +316,7 @@ export const getRowMetaData = (
 const formatStatsData = ( site: Site ) => {
```

Comment:
There's an issue with line 305 above:
```
client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts(305,3): error TS2322: Type 'SiteNode | StatsNode | BackupNode | ScanNode | PluginNode | MonitorNode' is not assignable to type '{ value: any; status: string; error?: boolean | undefined; }'.

[20:08:41 ](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/9641568?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&expandBuildChangesSection=true&showLog=9641568_41710_41710&logView=flowAware)
      Property 'value' is missing in type 'StatsNode' but required in type '{ value: any; status: string; error?: boolean | undefined; }'.
```

I've rebased and have reused [portions of Yashwin's code](https://github.com/Automattic/wp-calypso/pull/73730) and somehow his passes the test properly?

Reply:
The answer is hidden in the error message itself.

> Property 'value' is missing in type 'StatsNode' but required in type '{ value: any; status: string; error?: boolean | undefined; }'.

It looks like the property `value` is missing from `StatsMode`, is this expected?

Reply:
I see why it's saying that.

```
	row: {
		value: Site | any;
		status: AllowedStatusTypes | string;
		error?: boolean;
	};
```
It says it should have a value property defined.

I have just 

```
export interface StatsNode {
	type: AllowedTypes;
	status: AllowedStatusTypes | string;
	data: SiteStats;
}
```

which doesn't have a value property, it has a data property.

Similarly, in the Boost PR, I defined that as:

```
export interface BoostNode {
	type: AllowedTypes;
	data: BoostData;
	status: AllowedStatusTypes | string;
}
```

and it doesn't have a `value` and there's no error message on that PR.

I guess I don't understand what's different between:

- This and [PR73730](https://github.com/Automattic/wp-calypso/pull/73730/files#diff-ea8d42135b7e2a5de0de0c2b62c1a66cdee7853759ff9142763b727fb15b8589) (Yashwin's Stats PR)
- This and [PR74176](https://github.com/Automattic/wp-calypso/pull/74176) (My Boost PR)

Neither have a `value` property. I must be overlooking something between all three PRs.